### PR TITLE
feat(launcher): add IgnoreCerts

### PR DESCRIPTION
**Pull request by gnuletik**
fixes https://github.com/go-rod/rod/issues/753

### CODE REVIEW

**Review by ysmood — CHANGES_REQUESTED** [review]

File: lib/launcher/launcher.go [hunk 1]
```diff
@@ -253,6 +257,39 @@ func (l *Launcher) Devtools(autoOpenForTabs bool) *Launcher {
...
+	if err != nil {
+		return nil, fmt.Errorf("x509.MarshalPKIXPublicKey: %w", err)
+	}
+
+	sum := sha256.Sum256(pubDER)
+	pin := make([]byte, base64.StdEncoding.EncodedLen(len(sum)))
+	base64.StdEncoding.Encode(pin, sum[:])
+
+	return pin, nil
+}
```
[the remark this answers is not in the document]
Comment (gnuletik) [reply lib/launcher/launcher.go]: done !

**Comment by ysmood** [comment]
Still have some lint issues.

File: cspell.json [hunk 2]
```diff
@@ -106,7 +106,10 @@
         "wsutil",
         "xlink",
         "XVFB",
-        "ysmood"
+        "ysmood",
+        "spki",
+        "spkis",
+        "staticheck"
```
Comment (ysmood) [inline cspell.json:112]:
```suggestion
        "staticcheck"
```

**Review by ysmood — CHANGES_REQUESTED** [review]

File: cspell.json (hunk 2 above)
Comment (gnuletik) [reply cspell.json]: done!

**Review by ysmood — APPROVED** [review]
LGTM!

**Pull request merged**